crates/swc_ecma_minifier/tests/terser/compress/switch/drop_case/output.js

```diff
@@ -1,4 +1 @@
-switch (foo) {
-    case "bar":
-        baz();
-}
+if ("bar" === foo) baz();
```

Comment:
Test references in `crates/swc_ecma_minifier/tests/terser` should not be updated unless it's really required.

Reply:
Seems like they're updated in [this terser pr](https://github.com/terser/terser/pull/1044). Should I update output.terser.js too?

Reply:
Oh, I see.
Then yes

Reply:
How should I update `output.terser.js`? Is there a ready made script?

Reply:
I believe it's better to update terser in dep and update all terser reference at a separate pr, as currently reference seems to be made by terser 5.9 which is released half a year ago.

Reply:
Yeah, I agree, and to update terser test refs...
Hmm... I think I replaced `compress.js` in terser (submodule, `/vendor/terser`) with `scripts/terser/compress.js` and invoked terser tests.

I modified `compress.js` to print files to swc test directory.